docs/whatsnew-2.0.rst

```diff
@@ -0,0 +1,16 @@
+What's New in Pyramid 2.0
```

Comment:
This document generally gets built when we are ready to go to press... not sure we want to add it in this PR.